tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -325,6 +333,12 @@
 				if ( input.value ) {
 					query.push( encodeURIComponent( input.name ) + '=' + encodeURIComponent( input.value ) );
 				} else {
+					if (
+						input.name === 'jpcrm-populate-crm-data' ||
+						input.value === 'jpcrm-populate-woo-data'
+					) {
+						query.push( encodeURIComponent( 'jpcrm' ) );
```

Comment:
>  It looks like this force-enables Jetpack CRM on the site

The addition of jpcrm as a query parameter was mainly to work with the existing CRM setup in Jurassic Ninja (eg here - https://github.com/Automattic/jurassic.ninja/blob/master/features/jetpack-crm.php#L29 )

> As far as I can tell, "zero-bs-crm" was added to JN in September 2020

That's very true, we don't really need the additional plugin option that exists below now

> and is what you check for below?

See the below comment too (it was checking for the prefix before the branch name, but having the second plugin option also called `zero-bs-crm` was a bonus as it could work for that too, but that isn't really needed here).